packages/cypress-commands/src/index.ts

```diff
@@ -0,0 +1,2 @@
+import './commands';
+import './queries';
```

Comment:
```suggestion
import './commands.js';
import './queries.js';
```
ESM Packages always must import via file extension